docs/Makefile

```diff
@@ -18,4 +18,5 @@ help:
 # "make mode" option.  $(O) is meant as a shortcut for $(SPHINXOPTS).
 %: Makefile
 	cp ../CHANGELOG.md source/
+	cp CONTRIBUTING.md source/
```

Comment:
Does this file currently exist? Should this be `../CONTRIBUTING.md` if we are moving it to the root? Also I don't believe we currently have this file at the root level, so would you alsoneed to move that as part of this PR?

Reply:
> Does this file currently exist?

Yes, it's in `hi-ml/docs/`.

> Should this be ../CONTRIBUTING.md if we are moving it to the root?

If we do, yes. But I wanted to ask your opinion before moving it. Shall I?

Reply:
I see no reason not to. @ant0nsc?

Reply:
fine to move, sure.